lib/plugins/is-in-set-plugin.js

```diff
@@ -0,0 +1,18 @@
+/**
+ *
+ * @param varToCheck - a string which is checked whether it is contained in the set
+ * @param setWhichIncludesVar - an array of strings which may includes the varToCheck-String
```

Comment:
Kleiner Rechtschreibfehler, nicht weiter wichtig:
... which may include the ...